### ISSUE TITLE
try to get rid of nnf ... if possible

### ISSUE THREAD

**Issue by skeydan** [mlverse/torch#140]
Going to use `nnf_mse_loss` for the first time. In the context of a straightforward little MLP, this name for the loss function feels too convoluted I think ... wondering whether we could get rid of `nnf` completely.

Options I can think of:

1. (most consistent way)

use `f_`, as the Python guys do (`import torch.nn.functional as F`)

2. (less consistent, most intuitive [perhaps])

have losses start with `loss_` , activations with `activation_`, others with TBD (perhaps `f_` for the functions that correspond to layers?)

**Comment by skeydan** [comment]
Noticed I was looking for the equivalent of `torch.nn.MSELoss`, which I now found actually is `torch_mse_loss`.

I guess that makes option (2) obsolete, and - perhaps - this whole issue less urgent. I'm still keeping it open though as we might still want to think about (1).

**Comment by skeydan** [comment]
Hm, another thought. We probably want to replace the `torch` in `torch_mse_loss` by `nn`, which is more precise?

**Comment by skeydan** [comment]
Like we said,

(1) we'll have both classes and functions for loss etc.

(2) we replace `nnf_` by `f_`

(3) classes would look like `nn_mse_loss`

**Comment by dfalbel** [comment]
Finally let's stay with `nnf_`
